pkg/cmd/system.go

```diff
@@ -41,6 +47,8 @@ var (
 	settings = cli.New()
 )
 
+const definitionManifestDirectory string = "./config/samples"
```

Comment:
We should make built-in config as go code `https://github.com/cloud-native-application/RudrX/blob/master/pkg/builtin/deployment.go#L3`

Users will only install our cli binary, without anything else

Reply:
Fixed.